test/libjulia/Parser.cpp

```diff
@@ -269,6 +269,19 @@ BOOST_AUTO_TEST_CASE(multiple_assignment)
 	BOOST_CHECK(successParse(text));
 }
 
+BOOST_AUTO_TEST_CASE(if_statement)
+{
+	BOOST_CHECK(successParse("{ if 42:u256 {} }"));
```

Comment:
Hmm, shouldn't in Julia this operate on boolean? `if true:bool`. It currently has a hidden behaviour of `if iszero(cond) {}`

Reply:
I can change the test, but we do not yet enforce types anyway, right?

Reply:
No, not yet. It is fine to leave as it is, we'll at least see if the enforcement triggers this failure :)

Reply:
I see you've updated it, it is fine that way too.